app/views/dashboard/_volunteer_filters.html.erb

```diff
@@ -31,6 +31,43 @@
         <li><a class="small" data-value="option1" tabIndex="-1"><input style="margin:0 8px;" type="checkbox" data-value="No" checked />No</a></li>
       </div>
     </div>
+    <button type="button" class="btn btn-primary pull-right" data-toggle="modal" data-target="#exampleModal">
+      Pick displayed columns
+    </button>
   </div>
 </div>
 <br>
+
+<div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLabel">Pick Displayed Columns</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <div>
+					Select columns:
```

Comment:
This spacing feels odd, since spacing doesn't show up in the html. Not a big deal